web/packages/core/src/ruffle-player.ts

```diff
@@ -808,6 +808,53 @@ export class RufflePlayer extends HTMLElement {
             console.error("SWF download failed");
         }
     }
+    private openVirtualKeyboard(): void {
+        const virtualKeyboardId = "ruffle-virtual-keyboard";
+        const existingInput = this.shadow.getElementById(virtualKeyboardId);
+
+        if (existingInput) {
+            existingInput.focus({ preventScroll: true });
+            return;
+        }
+
+        const initialValue = "Initial";
+        const initialValueLength = initialValue.length;
+        const input = document.createElement("input");
```

Comment:
Maybe define an `<input>` element statically in `shadow-template.ts`? Now it seems like the dynamically-created elements are "leaked" into the DOM (i.e. added but never removed).

Reply:
Sorry, I've been busy with the iOS PR. I agree with your idea here, and it would probably be simpler to do that. Technically the input will exist on desktop too and just never be used, whereas here it only exists on touch devices since it only appears after choosing the context menu option, but that's probably fine. I'll try to implement this.

Reply:
I imagine something like:
```css
#ruffle-virtual-keyboard {
  position: absolute;
  opacity: 0;
  top: -100px;
  width: 1px;
  height: 1px;
}
```
```html
<input id="ruffle-virtual-keyboard" type="text" autocapitalize="off" autocomplete="off" autocorrect="off" value="Initial" oninput="virtualKeyboardInput()">
```